sqlite3/sqlite3.php

```diff
@@ -256,7 +256,7 @@ public function createFunction(
         #[LanguageLevelTypeAware(['8.0' => 'string'], default: '')] $name,
         #[LanguageLevelTypeAware(['8.0' => 'callable'], default: '')] $callback,
         #[LanguageLevelTypeAware(['8.0' => 'int'], default: '')] $argCount = -1,
-        int $flags = 0
+        #[PhpStormStubsElementAvailable(from: '7.1')] int $flags = 0
```

Comment:
Default value for `$flags` is available only starting PHP 8.0.

Reply:
Yep, but still it was optional in earlier versions and default value here is a way to mark it as optional